RELEASING.md

```diff
@@ -238,6 +251,10 @@ as described in the section above.
    git add _data/release-notes/<VERSION>-release-notes.md
    ```
 
+1. In [netlify.toml](netlify.toml):
+      
+      1. Set the `RELEASE_VERSION` environment variable to new version.
```

Comment:
Do we still need this step since we already do this on the branch when the release branch is created?